src/Runner.Common/Constants.cs

```diff
@@ -223,6 +223,7 @@ public static class Actions
             public static class Agent
             {
                 public static readonly string ToolsDirectory = "agent.ToolsDirectory";
+                public static readonly string ForcedNodeVersion = "GITHUB_ACTIONS_RUNNER_FORCED_NODE_VERSION";
```

Comment:
Where else should we document this var? Release notes?

Reply:
we don't document these, we will let customers know if they run into issues.